src/features/apiDelegation/offered/ChooseApiPage/ChooseApiPage.tsx

```diff
@@ -75,6 +70,7 @@ export const ChooseApiPage = () => {
         }
       });
     }
+    setDelegationCheckLoading(false);
```

Comment:
This needs to be set after we've received the results from the delegationChecks. Right now, the skeletons disappear before the changes happen on screen, which means that we see the actionBars go from one side to the other.

Reply:
I guess that means we have to either put this state into the redux state or make delegationCheck into a rtkQuery api

Reply:
Ahh yeah you're right. Best thing would be if we could await the last call on delegationCheck or something. Thinking about it, that's something we maybe could have done if delegationCheck was a mutation in RTK Query instead of being a slice? I Haven't analyzed it yet, but it may work. If that's something you want to try out when I'm on vacation you can just take over this whole task as you please 😊

Reply:
Yeah, since this will soon go into production, I think I'll sneak into this branch and try to implement delegationCheck with rtkQuery and mutations 😊